Gemfile

```diff
@@ -12,6 +12,7 @@ gem "mechanize"
 gem "sitemap-parser", github: "fastruby/sitemap-parser",
                       branch: "fixes/recurse-to-level-n"
 gem "ombu_labs-shortener"
+gem "ombu_labs-auth", "1.0.0"
```

Comment:
@arielj Do we want to use fixed versions here? This will need to be changed if we release 1.0.x and might be impractical. Thoughts?

Reply:
for some reason it was installing `0.1.0` for me, I had to specify 1.0.0

I can remove it now (it's in the lock file anyway)

Reply:
done :+1: